README.md

```diff
@@ -16,4 +16,19 @@ Just click here to start:
 
 ## Python
 
-Tutorials in [python](https://www.python.org) programming language for [reforms](http://openfisca.org/doc/reforms.html) creation on the French tax and benefit system.
+### [Python tutorials for OpenFisca-France](./python/)
+
+With the Python [Tutorials](./python/) (in French), you will learn how to:
+- use the python API to run calculations
+- get results on the current legislation as well as on reforms
+- export calculation result to CSV
+
+### [Situations for OpenFisca-France](./python/situations)
+This directory contains pre-filled situations extracted from the French "Livret du pouvoir d'achat".
+
+You can use them 'as is' in the web API or import them in the python API as shown in the python tutorials. 
+
+### [Python scripts for OpenFisca-France](./python/scripts)
```

Comment:
> Ajouter la ligne de commande pour les exécuter ? Ex :
To run one of the following scripts, go to its directory an call it with `python` command.